sos/report/plugins/snap.py

```diff
@@ -28,4 +29,15 @@ def setup(self):
         self.add_service_status("snapd")
         self.add_journal(units="snapd")
 
+    def postproc(self):
+        self.do_file_sub(
+            "/var/lib/snapd/state.json",
+            r"\"(macaroon|store-macaroon|key-id|session-macaroon"
+            + "|macaroon-key)\":\"([A-Za-z0-9_=-]*)\"",
```

Comment:
I believe this also needs to be explicitly a raw string, given the escapes in it. You can also use normal string concatenation here.

```python
            (r"\"(macaroon|store-macaroon|key-id|session-macaroon"
             r"|macaroon-key)\":\"([A-Za-z0-9_=-]*)\""),
```

Reply:
Changes done; thanks for catching that and the suggestion!